### PR TITLE
Discontinue support (in Student Finance Form) for Part Time EU students in year 2014/2015

### PR DESCRIPTION
**Pull request by ikennaokpala**
Supersedes #2515 

[Trello Story](https://trello.com/c/Fy7pU4Dy/112-student-form-finder)

Part-time students can no longer apply for 2014/15; they can apply for 2016/17. 

This update removes the remaining 14/15 outcomes and adds some for 16/17. Logic will require update to match this - 15/16 and 16/17 should be the only date choices. 

Changes to go live morning of Monday 16 May. 

## Factcheck
[Preview Link]( https://smart-answers-pr-2517.herokuapp.com/student-finance-forms/y)
[GOV.UK]( https://gov.uk/student-finance-forms/y)

### Expected changes

* Selectable options and outcomes respectively should not contain any 2014-2015 year information only 2015 to 2016.

### Before

screen shot 2016-05-10 at 17 09 07

### After

screen shot 2016-05-10 at 17 09 34

### CODE REVIEW

**Comment by stwalsh** [comment]
Updated version looks good to me - thanks!
Sean

**Comment by ikennaokpala** [comment]
@stwalsh thanks for letting us know..

File: test/data/student-finance-forms-files.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ lib/smart_answer_flows/student-finance-forms/outcomes/_where_to_send_your_forms_
 lib/smart_answer_flows/student-finance-forms/outcomes/outcome_ccg_1516.govspeak.erb: 8c4197c95ee1efd165da56a007540829
```
Comment (pmanrubia) [inline test/data/student-finance-forms-files.yml]: Is there any reason why you are including this file?

**Comment by pmanrubia** [comment]
Please, update or clarify why the file is included in that commit.
Other than that: LGTM

File: test/data/student-finance-forms-files.yml (hunk 1 above)
Comment (ikennaokpala) [inline test/data/student-finance-forms-files.yml]: Not intentional.

**Pull request merged**